NitroxServer/Communication/UdpServer.cs

```diff
@@ -83,6 +101,10 @@ private void Listen()
 
         private void ProcessIncomingData(Connection connection, byte[] data)
         {
+#if SWPF
```

Comment:
Not a blocker but maybe you're interested nonetheless. 

Things like this would be better placed inside a wrapper class that wraps the UdpServer using the decorator pattern so you can switch WPF on or off at runtime by decorating the UdpServer class with additional logic like keeping track of bytes sent and it keeps the UdpServer class's code intact, lean and clean.

In short on how to implement the decorator pattern for this pr: 
1. You define an interface for the UdpServer with all the properties/functions you need.
2. Create an _abstract_ wrapper class that implements the interface and contains a private set property of the interface's type that can be given through the wrapper's constructor. So the wrapper _is_ and _has_ the interface. This is a **very simple** class that just passes all functions through as virtual.
3. Create the class that extends the abstract wrapper to add your profiling logic, overriding the functions it needs or wants. A good class name might be `UdpServerWpfProfiler`
4. Then you can switch UdpServer functionality at runtime: `IUdpServer Server = new UdpServerWpfProfiler(new UdpServer());` or just do new UdpServer() and you get a server that doesn't report any information.

If you like to learn more or just refresh your memory then this is a good video: https://youtu.be/GCraGHx6gso?t=12m50s.